mbed_lstools/main.py

```diff
@@ -204,6 +207,9 @@ def parse_cli(to_parse):
         '-m', '--mock', metavar='ID:NAME',
         help='substitute or create a target ID to platform name mapping used'
         'when invoking mbedls in the current directory')
+    commands.add_argument(
```

Comment:
maybe here could be also optional update url argument that is used to fetch updates - with default value which are visible in help..

Reply:
Definitely. Very good spot!